### PR TITLE
simple cases for trying Stream API in action

### PR DESCRIPTION
**Pull request by Yuri13579**

### CODE REVIEW

**Comment by Yuri13579** [comment]
simple cases for trying Stream API in action

File: src/main/java/co/inventorsoft/StreamHomework.java [hunk 1]
```diff
@@ -38,8 +43,11 @@
      * @param users collection of users
      * @return map {user email : user}
      */
-    public Map<String, User> groupByEmail(final List<User> users) {
-        return null;
+    public Map<String, User> groupByEmail(final List<User> users)
+    {
```
Comment (stefankiv) [inline src/main/java/co/inventorsoft/StreamHomework.java]: Let's stick to java code conventions and always put opening curly brace on the same line with method name.

**Review by stefankiv — CHANGES_REQUESTED** [review]
Nice :+1: 
Please, work a little bit on code formatting. I'd recommend to put each stream operator on a new line. Also pay more attention to spaces between operators **+ - / * =**

File: src/main/java/co/inventorsoft/StreamHomework.java (hunk 1 above)
Comment (Yuri13579) [reply src/main/java/co/inventorsoft/StreamHomework.java]: public Map<String, User> groupByEmail(final List<User> users) {
        return users.stream()
                .collect(Collectors.toMap(User::getEmail, Function.identity()));

    }

**Review by stefankiv — APPROVED** [review]
let it be :+1:

**Pull request closed without merging**